docs/Files/UI/Solution-Manager/Library/Managing-Global-Properties.md

```diff
@@ -0,0 +1,13 @@
+---
+lang: en-us
+title: Managing Global Properties
+viewport: width=device-width, initial-scale=1.0
+---
+
+# Overview
+
+The Global Properties editor also has a toolbar that allows you to perform functions for managing the global properties. Learn to add page to Add or Edit global properties, add user defined system properties.
+
+![Managing Library](../../../../Resources/Images/SM/Library/ManagingLibrary/ManagingGlobalProperties.png "Threshold Grid")
```

Comment:
we should update the alt text in links. for example, this should not be "Threshold Grid"

Reply:
fixed all over